javascript/javascript1/week1/app.js

```diff
@@ -0,0 +1,78 @@
+// Age-ify (A future age calculator)
+let yearOfBirth = 1987;
+let yearFuture = 2027;
+let age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
```

Comment:
Clean and clear code 👌 
Which one of these variables would most definitely be a const? 😊